src/math_util.c

```diff
@@ -1050,65 +1051,67 @@ s32 func_802B8058(u16 arg0, u16 arg1, u16 arg2) {
     return 1;
 }
 
-//generated by m2c commit 3b40ab93768f52ac241c5ae84ef58ef6bc4cb1de
-f32 func_802B80D0(Vec3f arg0, Vec3f arg1, u16 arg2, f32 arg3, f32 arg4, f32 arg5) {
-    u16 temp_a2;
+/**
+ * Returns distance or -1.0f if exceeds render distance.
+*/
+f32 is_within_render_distance(Vec3f cameraPos, Vec3f objectPos, u16 orientationY, f32 minDistance, f32 fov, f32 maxDistance) {
```

Comment:
I think you might have forgotten to rename every instance of some renames in the repo

You may find it easier to use the ./rename_sym old_name new_name command

Reply:
Also found gFlatBananaVertices and FlatBanana